model/CHEM_DRV.F90

```diff
@@ -608,6 +608,12 @@ SUBROUTINE DO_CHEM
     ENDIF
 
     IF ( FIRST_CHEM ) THEN
+
+       ! Set species units to kg kg-1
+       DO N=1, State_Chm%nSpecies
+         State_Chm%Species(N)%Units = KG_SPECIES_PER_KG_DRY_AIR
+       ENDDO
```

Comment:
Try printing units before this to check what they are. Should be set by GEOS-Chem when this is being read in. Input file itself is $mol\,mol^{-1}$.

Reply:
On Friday I put in some debug prints to check what the species' units are at the start of the `FIRST_CHEM` conditional section in `model/CHEM_DRV.F90`. I just got
```
...
[0]  DEBUG:    Species           1           0
[0]  DEBUG:    Species           2           0
...
[0]  DEBUG:    Species         310           0
[0]  DEBUG:    Species         311           0
...
```
indicating that the units are unset.